package.json

```diff
@@ -2308,15 +2308,11 @@
                     "default": "docker",
                     "description": "%vscode-docker.config.docker.dockerPath%",
                     "scope": "machine-overridable"
-                }
-            }
-        },
-        "configurationDefaults": {
-            "[dockercompose]": {
```

Comment:
This may need to come back

Reply:
I don't think so. It is controlling always-on completions which are not viable (yet).